src/freenet/support/Fields.java

```diff
@@ -76,12 +76,12 @@ public void shouldUpdate(){
 		'z'
 	};
 	private static final long[] MULTIPLES = {
-		1000, 1l << 10,
-		1000 * 1000, 1l << 20,
-		1000l * 1000l * 1000l, 1l << 30,
-		1000l * 1000l * 1000l * 1000l, 1l << 40,
-		1000l * 1000l * 1000l * 1000l * 1000, 1l << 50,
-		1000l * 1000l * 1000l * 1000l * 1000l * 1000l, 1l << 60
+		1000, 1L << 10,
+		1000 * 1000, 1L << 20,
+            1000L * 1000L * 1000L, 1L << 30,
+            1000L * 1000L * 1000L * 1000L, 1L << 40,
+            1000L * 1000L * 1000L * 1000L * 1000, 1L << 50,
+            1000L * 1000L * 1000L * 1000L * 1000L * 1000L, 1L << 60
```

Comment:
indentation

Reply:
I am unable to find an issue with the indentation of the source file.